transcendence-app/src/user/user.controller.ts

```diff
@@ -168,19 +168,30 @@ export class UserController {
     return avatar;
   }
 
+  @Get('block/list')
```

Comment:
We're not using this endpoint at the moment, are we?

What do you think we will need it for?